python/lopq/model.py

```diff
@@ -394,7 +399,7 @@ def train(data, V=8, M=4, subquantizer_clusters=256, parameters=None,
 
 
 class LOPQModel(object):
-    def __init__(self, V=8, M=4, subquantizer_clusters=256, parameters=None):
+    def __init__(self, V=8, M=4, subquantizer_clusters=256, n_jobs=1, parameters=None):
```

Comment:
Let's remove this as an initializer argument and as an attribute of the model class - `n_jobs` is really an attribute of the fitting process, not the model itself. It should be an argument to `LOPQModel.fit`.

Reply:
I am relying on the well established Scikit-Learn standard standard that normally attributes this argument to the specific model and not the fitting process. Therefore, I relied on this widely used an therefore common way to model it. Nevertheless, I agree with you, it is rather fitting process as model specific and gives you more flexibility. I am fine with it.

Reply:
Yeah, good point. Perhaps it would have been better for this project to conform to that convention at the beginning, but at this point there is already the division between `fit` parameters and `__init__` parameters.